### ISSUE TITLE
Amlogic S905X4 Android 11 box, no sound when using internal dvb-c/t2 tuner

### ISSUE THREAD

**Issue by kalehrl** [AndroidAudioMods/ViPER4Android#67]
Hi
I've been using Viper4Android on an Amlogic S905X4 Android 11 Google certified box and it mostly works fine. However, when watching channels via the built-in dvb-c/t2 tuner, there is no sound at all. I have to disable Viper4Android for the sound to return. Any ideas how it might be fixed?

**Comment by iscle** [comment]
Hi,

The tuner is probably using another audio path or something. Can you send an screenshot of the driver status dialog?

**Comment by kalehrl** [comment]
You mean this one?
![screencap](https://github.com/AndroidAudioMods/ViPER4Android/assets/3274489/72a650b6-ee1e-471d-b775-fc3e7ec66e0a)

**Comment by iscle** [comment]
Yes. It looks like you're not using this project, but an old one which is completely different.

I can't help you in that case. If you want to use this project, check the Releases page :)

**Comment by kalehrl** [comment]
I'm using this one:
https://github.com/programminghoch10/ViPER4AndroidRepackaged
I thought that's the latest one.
I will try this one but I'm not sure if I will be able to add irs files because Android 11 has restricted /sdcard/Android/data access.

**Comment by kalehrl** [comment]
![screencap](https://github.com/AndroidAudioMods/ViPER4Android/assets/3274489/59e14374-4b48-44ef-ac8f-6bba6873778f)
Unfortunately, I can't import kernel because there is a message that reads "You don't have an app that can do this"

**Comment by iscle** [comment]
And Convolver is not supported yet in this version, only the non greyed out options :)

**Comment by kalehrl** [comment]
No sound with this version as well :(

**Comment by iscle** [comment]
The screenshot is of when the tuner is active (and audio not working) or when the audio does work?

**Comment by kalehrl** [comment]
To start Viper4Android I have to exit Live Channels app which I use for watching cable channels. I can't make screenshot and watch channels at the same time.

**Comment by iscle** [comment]
Hm... I guess I'll have to implement some sort of log in that case. In a future version :)

**Comment by mpeter50** [comment]
@kalehrl cant you use split screen?

I dont know Android 11, but on 14 you go to the recents menu (square button on the navigation bar), tap the icon of an app at the top, and choose "split screen", then you can open another app.
That way you may be able to make a screenshot of the Viper status.

![image](https://github.com/user-attachments/assets/7470b5d9-c658-48d8-beab-61d8f33244da)

If you dont have the split screen option, maybe you can turn it on or something similar in the developer options, but be careful with those.
